zproject/dev_urls.py

```diff
@@ -14,6 +14,10 @@
 use_prod_static = getattr(settings, 'PIPELINE_ENABLED', False)
 
 urls = [
+    # Serve node_modules for code context in blueslip's stacktrace
+    url(r'^node_modules/(?P<path>.*)$',
+        serve, {'document_root': os.path.join(settings.DEPLOY_ROOT, 'node_modules')}),
```

Comment:
Hold on. Doesn’t the source map already contain all the code? Why do we need to fetch it separately?